README.md

```diff
@@ -136,8 +160,10 @@ egrep -Rin "TODO|FIXME" -R --exclude-dir=vendor *
 
 * [Babiel](https://babiel.com) (supporter)
 
-* Author on Twitter [@jptosso](https://twitter.com/jptosso)
+### Author on Twitter 
```

Comment:
What do you think about creating a community Twitter account?

Reply:
I think it is a good idea!